### PR TITLE
LibAnswers - Moved custom styles repeated on every FAQ Group page to central styles.

### PR DESCRIPTION
**Pull request by djanelle-mit**
### Why these changes are being introduced

When creators were adding new FAQ Group pages in LibAnswers, they were copying and pasting custom styles from a previous example into their new page. This was done on nearly all public-facing pages.

### How this addresses that need

I centralized the custom styles into the system styles, and prepended classes for the three page types (Home, FAQ, Search) that were affected by the custom styles. This should avoid any downstream affects now that this custom styling is added in a central location.

Note: I also added a file that preserves the old styling that was added to each page for posterity since there were some style rules that were commented out. I wasn't sure if we'd need them in the future so opted to preserve. Happy to adjust if needed.

### Side effects of this change

n/a

### Relevant ticket(s)
https://mitlibraries.atlassian.net/browse/UXWS-1670

### CODE REVIEW

**Review by jazairi — APPROVED** [review]
Sorry it took so long to get a review on this! These changes make sense to me and seem like they should make content creators' lives easier. Nice work!

I'm ambivalent about checking the old styling into version control. I think if there's a possibility that we may need it in the future, then it's probably wise to do so. There may be a small risk that the file could confuse future-us, but I'm not particularly worried about that since you've explained why it's there.

Speaking of the PR message, our practice in EngX is to include that extended detail in our commit message. [Here's an example](https://github.com/MITLibraries/tacos/pull/127/commits/e3a6b7b9cf0b5f9f2bc190e48520a50d7ae410ad). The intent is to make it easier to understand the full context for a change when digging through git history.

Totally up to you as to whether you want to adopt this practice, but if you do, I recommend setting up a [commit message template](https://gist.github.com/lisawolderiksen/a7b99d94c92c6671181611be1641c733). It makes it much easier not to have to type in the headings each time.

Thanks for taking this on, and sorry again for the delayed review!

**Comment by djanelle-mit** [comment]
@jazairi I wavered on keeping the old code in there and ended up stripping it out. I think it would end up causing us more confusion down the road.

On the commit message topic: some of these PRs have had a lot of commits, and I think I probably need to wait for full approval before committing anything, but in ideal cases where the changes are small, do you find the detailed commit message template helpful in addition to the PRs message? I can totally see it being helpful for larger chunks of work.

I'm happy to adopt if it adds clarity for other folks, too. For larger chunks of work where each commit has more to it, I could totally see the value, too.

**Comment by djanelle-mit** [comment]
@jazairi No worries on the delay, hope you're feeling better now! 

Awesome! I'll experiment a little with finding that threshold and see what feels like the right amount of detail. Always happy for feedback as we go on what works best!

**Pull request merged**
